### PR TITLE
Add Vertical, SupplySource, AudienceSegment entities

### PR DESCRIPTION
**Pull request by FodT**

### CODE REVIEW

File: terminalone/models/audiencesegment.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        'parent_audience_segment_id': int,
+        'retail_cpm': float,
+        'wholesale_cpm': float,
+        'tag': None,
+        'uniques': int,
+        'updated_on': Entity._strpt,
+        'version': int,
+    }
+
+    _push = _pull.copy()
```
Comment (pswaminathan) [inline terminalone/models/audiencesegment.py]: If `_push` and `_pull` are the same, we don't need the copy. Just an alias. The copy is only needed if we want to modify `_push`.

File: terminalone/models/audiencesegment.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""Provides audience segment object."""
+
+from __future__ import absolute_import
+from ..entity import Entity
+
+
+class AudienceSegment(Entity):
+    """docstring for AudienceSegment"""
```
Comment (pswaminathan) [inline terminalone/models/audiencesegment.py]: Let's make this a non-placeholder docstring, even if it is just something like `"""Audience segment object"""`

File: terminalone/models/supplysource.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        'rtb_enabled': Entity._int_to_bool,
+        'rtb_type': None,
+        'seat_enabled': Entity._int_to_bool,
+        'status': Entity._int_to_bool,
+        'supply_type': None,
+        'updated_on': Entity._strpt,
+        'use_pool': Entity._int_to_bool,
+        'version': int,
+    }
+    _push = _pull.copy()
```
Comment (pswaminathan) [inline terminalone/models/supplysource.py]: Same thing with here with copy.

File: terminalone/models/vertical.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    _pull = {
+        'id': int,
+        'name': None,
+        'created_on': Entity._strpt,
+        'updated_on': Entity._strpt,
+        'version': int,
+    }
+    _push = _pull.copy()
+    _push.update({
+        'name': None,
```
Comment (pswaminathan) [inline terminalone/models/vertical.py]: Here we don't even need the update, as `name` is the same as in `_pull`

File: terminalone/models/supplysource.py (hunk 3 above)
Comment (FodT) [inline terminalone/models/supplysource.py]: Looks like we actually need to add all the bool push types

File: terminalone/models/audiencesegment.py (hunk 1 above)
Comment (FodT) [inline terminalone/models/audiencesegment.py]: i missed 'buyable' push type

**Pull request merged**